section-09-differential-testing/model-package/requirements/test_requirements.txt

```diff
@@ -0,0 +1,13 @@
+-r requirements.txt
+
+# testing requirements
+pytest>=6.2.3,<6.3.0
+
+# repo maintenance tooling
+black==20.8b1
+flake8>=3.9.0,<3.10.0
+mypy==0.812
+isort==5.8.0
+
+# new in section 7: for fetching data in CI
+kaggle>=1.5.12,<1.6.0
```

Comment:
Are we still fetching from kaggle? I thought we removed it because it is not a realistic scenario.

Reply:
We're fetching from kaggle so we don't have to keep the `test.csv` and `train.csv` in the repo